package.json

```diff
@@ -21,7 +21,8 @@
     "node-sass": "^4.12.0",
     "react": "~16.8.1",
     "react-dom": "~16.8.1",
-    "react-scripts": "2.0.5"
+    "react-scripts": "2.0.5",
+    "prop-types": "latest"
```

Comment:
can we rather use a version number: `latest` can introduce breaking changes

Reply:
allianz and assa
```
"prop-types": "^15.6.0"
```

Reply:
done